proto/dymensionxyz/dymension/rollapp/params.proto

```diff
@@ -2,31 +2,42 @@ syntax = "proto3";
 package dymensionxyz.dymension.rollapp;
 
 import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+
 
 option go_package = "github.com/dymensionxyz/dymension/v3/x/rollapp/types";
 
 message DeployerParams {
   // address is a bech32-encoded address of the 
   // accounts that are allowed to create a rollapp.  
-  string address = 1 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+  string address = 1 [(gogoproto.moretags) = "yaml:\"address\""];
 }
 
 // Params defines the parameters for the module.
 message Params {
   option (gogoproto.goproto_stringer) = false;
-  
+
   // dispute_period_in_blocks the number of blocks it takes
   // to change a status of a state from received to finalized.
   // during that period, any user could submit fraud proof
   uint64 dispute_period_in_blocks = 1
-      [ (gogoproto.moretags) = "yaml:\"dispute_period_in_blocks\"" ];
+  [(gogoproto.moretags) = "yaml:\"dispute_period_in_blocks\""];
 
   // deployer_whitelist is a list of the
   // accounts that are allowed to create a rollapp and maximum number of rollapps.
   // In the case of an empty list, there are no restrictions
   repeated DeployerParams deployer_whitelist = 2
-      [ (gogoproto.moretags) = "yaml:\"deployer_whitelist\"" , (gogoproto.nullable) = false];
+  [(gogoproto.moretags) = "yaml:\"deployer_whitelist\"", (gogoproto.nullable) = false];
 
   bool rollapps_enabled = 3
-      [ (gogoproto.moretags) = "yaml:\"rollapps_enabled\"" ];
+  [(gogoproto.moretags) = "yaml:\"rollapps_enabled\""];
+
+  // How long it typically takes to produce a Hub block (e.g. 6 secs). Used to calculate length of time based on height.
+  google.protobuf.Duration hub_expected_block_time = 4 [(gogoproto.moretags) = "yaml:\"hub_expected_block_time\"", (gogoproto.nullable) = false, (gogoproto.stdduration) = true];
```

Comment:
OK I don't really have a opinino on this
would have been good to have this discovered at ADR stage tbh since the params were written in the ADR like two weeks ago already

Reply:
then it would probably be a good idea to have a review/approve process for ADRs prior to the implementation

Reply:
Omri and Michael did review
but yes we need to do better on it

Reply:
anyways, done